macros/edr/tests/test_utils/get_anomaly_query.sql

```diff
@@ -36,11 +37,20 @@
       select
         metric_value as value,
         training_avg as average,
-        case when is_anomalous = TRUE then
+        -- when there's an anomaly we would want to use the last value of the metric (lag), otherwise visually the expactations would look out of bounds
```

Comment:
dbt uses `--` comments when parsing, so the BP here is to use the Jinja comments syntax `{# #}`